### PR TITLE
Fix numbers in translations

### PR DESCRIPTION
**Pull request by Blake-Madden**
Mostly just paper sizes and code pages

### CODE REVIEW

File: locale/ms.po [hunk 1]
```diff
@@ -2655,7 +2655,7 @@ msgstr "Helai A4, 210 x 297 mm"
 
 #: ../src/common/paper.cpp:95
 msgid "C sheet, 17 x 22 in"
-msgstr "Helai D, 22 x 34 in"
+msgstr "Helai D, 17 x 22 in"
```
Comment (lanurmi) [inline locale/ms.po:2658]: Here presumably also D should be C.

Comment (Blake-Madden) [reply locale/ms.po]: Got it, thanks.

**Comment by vadz** [comment]
Nice, thanks! Did you find all of them on your own or do you have a tool detecting this kind of errors?

**Comment by Blake-Madden** [comment]
I'm working on a tool to detect i18n issues like this...built with wxWidgets, of course :)

**Pull request merged**